content/general/authentication.textile

```diff
@@ -226,176 +235,59 @@ auth.requestToken({ capability: {
 
-* the @keyName@ comprising of the app ID and key ID such as @{{API_KEY_NAME}}@
-* a capability (i.e. a set of channel names/namespaces and, for each, a set of "operations":#capability-operations) which should be a subset of the set of capability associated with the key specified in key ID
-* optionally, a @clientId@ thus identifying clients using this token and preventing them from identifying themselves with any other @clientId@
-* optionally, an expiry time or TTL, will default to 1 hour if not specified
-* a timestamp to ensure token request is still valid
-* a unique nonce string, randomly-generated by the client, to ensure the token request cannot be reused
-
-A signed token request also contains:
-
-* a signature, generated as an HMAC of each of the above components, using the key secret value.
-
-Signed token requests can be used to request a token from Ably without an authenticated connection. The signature generated with the key secret confirms the authenticity of the token and can thus be "trusted" by Ably. As signed token requests can be issued without a request to Ably, a server with a valid API key can issue token requests directly to clients, and clients can in turn generate a token by sending the token request to Ably.
-
-The receiving Ably server verifies the signature if present, the timestamp (which must be within 2 minutes of the current time), verifies that the nonce/timestamp combination has not been used previously, verifies that the requested validity period is permitted, and verifies that the requested capabilities are permitted for that token based on the key capabilities.
-
-The server may choose to subset the capabilities based on the capabilities of the key.
+Given an API key exists with the following capabilities:
 
-The server replies with an access token, which is essentially a signed version of the resolved set of capabilities, plus other metadata associated with the token (such as expiry time).
+```[json]
+{
+  "chat:team:*": ["*"]
+}
+```
 
-This access token can then be used for subsequent REST requests or Realtime connections. If a @clientId@ was included in the request, then the token is associated with that @clientId@, and may be used to identify that client in operations that require identification (e.g. joining a channel that requires identification, or publishing a message with a verified @clientId@).
+And a token is requested with the following explicit capabilities:
 
-h3. Parameter canonicalisation
+```[javascript]
+auth.requestToken({ capability: {
+  "chat:*": ["*"]
+}}, tokenCallback)
+```
 
-The parameters of the token request are normalized/canonicalized as follows:
+Then Ably will be unable to issue a token because the requested channel scope is greater than the API key's capabilities i.e. @chat:*@ is requested yet the API key only has acccess to the narrower @chat:team:*@ channel scope. In the example above, @requestToken@ will call the callback with an error.
```

Comment:
> `requestToken` will call the callback with an error.

might be worth clarifying that this is only because the resulting token would have literally no capabilities. If the token requests two things and one is satisfiable, it'll happily issue a token with just that one